recipes/libboost-mpi/conda_build_config.yaml

```diff
@@ -0,0 +1,9 @@
+mpi:
+  - mpich    # [unix]
+  - openmpi  # [unix]
+  - msmpi    # [win64]
+
+pin_run_as_build:
+  mpich: x.x
+  openmpi: x.x
+  msmpi: x.x
```

Comment:
```suggestion
```

Reply:
For my records, this is not necessary because it's the default pinning behavior?